src/main/java/listeners/VideoRecorder.java

```diff
@@ -19,53 +20,47 @@
+    private static final Integer SCREEN_FRAME_RATE = 15;
+    private static final Integer SCREEN_DEPTH_VALUE = 24;
+    private static final String MOUSE_COLOR = "black";
+
     private ScreenRecorder screenRecorder;
 
     public void startRecording() {
         try {
-            GraphicsConfiguration gc = GraphicsEnvironment
-                    .getLocalGraphicsEnvironment().getDefaultScreenDevice()
-                    .getDefaultConfiguration();
-            File dir = new File(RECORD_DIRECTORY);
+            GraphicsConfiguration graphicsConfiguration = GraphicsEnvironment.getLocalGraphicsEnvironment()
+                    .getDefaultScreenDevice().getDefaultConfiguration();
+            File file = new File(RECORD_DIRECTORY);
             Point point = DriverManager.getDriver().manage().window().getPosition();
             Dimension dimension = DriverManager.getDriver().manage().window().getSize();
-            Rectangle rectangle = new Rectangle(point.x, point.y,
-                    dimension.width, dimension.height);
-            this.screenRecorder = new ScreenRecorder(gc, rectangle,
-                    new Format(MediaTypeKey, MediaType.FILE, MimeTypeKey,
-                            MIME_AVI),
-                    new Format(MediaTypeKey, MediaType.VIDEO, EncodingKey,
-                            ENCODING_AVI_TECHSMITH_SCREEN_CAPTURE,
-                            CompressorNameKey,
-                            ENCODING_AVI_TECHSMITH_SCREEN_CAPTURE, DepthKey,
-                            24, FrameRateKey, Rational.valueOf(15), QualityKey,
-                            1.0f, KeyFrameIntervalKey, 15 * 60), new Format(
-                    MediaTypeKey, MediaType.VIDEO, EncodingKey,
-                    "black", FrameRateKey, Rational.valueOf(30)), null,
-                    dir);
+            Rectangle rectangle = new Rectangle(point.x, point.y, dimension.width, dimension.height);
+            this.screenRecorder = new ScreenRecorder(graphicsConfiguration, rectangle,
+                    new Format(MediaTypeKey, MediaType.FILE, MimeTypeKey, MIME_AVI), // add file format
+                    new Format(MediaTypeKey, MediaType.VIDEO, EncodingKey, ENCODING_AVI_TECHSMITH_SCREEN_CAPTURE,
+                            CompressorNameKey, ENCODING_AVI_TECHSMITH_SCREEN_CAPTURE, DepthKey, SCREEN_DEPTH_VALUE, FrameRateKey,
+                            Rational.valueOf(SCREEN_FRAME_RATE), QualityKey, QUALITY_VALUE, KeyFrameIntervalKey, FRAME_INTERVAL_VALUE), // add screen format
+                    new Format(MediaTypeKey, MediaType.VIDEO, EncodingKey, MOUSE_COLOR, FrameRateKey, Rational.valueOf(MOUSE_FRAME_RATE)), // add mouse format
+                    null, file); // add audio format and movie folder
             this.screenRecorder.start();
-        } catch (Exception e) {
-            e.printStackTrace();
+        } catch (IOException | AWTException e) {
+            throw new IllegalArgumentException(e.getMessage());
```

Comment:
Why is it IllegalArgumentException? 
Create a custom exception 'VideoRecorderException' and throw it here